frontend/src/screens/learner.tsx

```diff
@@ -21,23 +21,51 @@ const HighlightedStudies: FC = () => {
     const { highlightedStudies } = useParticipantStudies()
     const isMobile = useIsMobileDevice()
 
+    const handleClick = () => {
```

Comment:
Maybe something a bit more descriptive than the generic `handleClick` - something like `scrollToStudies` or something?

Reply:
Okay